models/cost_per_query.sql

```diff
@@ -6,7 +6,7 @@ daily_rates as (
         date,
         effective_rate
     from {{ ref('daily_rates') }}
-    where usage_type = 'compute'
+    where (usage_type = 'compute' or usage_type = 'overage-compute')
```

Comment:
```suggestion
    where usage_type in ('compute', 'overage-compute')
    group by date
```

This concerns me a little in the event that there's both a `compute` and an `overage-compute` entry for a day. I have no idea if that can happen. For now, let's just group by `date` and take the max rate?

Reply:
Eeish.  Good point.  My understanding is that 100% of spend is overage on payg plans, on a committed plan the overage will start accruing after the agreed spend has been used up.  I guess that means they will both happen on the same day for some clients.